lib/fluent/logger/fluent_logger.rb

```diff
@@ -153,23 +162,25 @@ def write(msg)
           if @pending
             @pending << data
           else
-            @pending = data
+            @pending = [data]
```

Comment:
Sorry. My idea is `@pending` should be Hash, `{'tag1' => "msgpack stream", 'tag2' => "msgpack stream", ...}`.
Using this structure, we can send multiple events in one TCP write.